tests/unit/components/tumblr-post-test.js

```diff
@@ -1,20 +1,52 @@
+import Ember from 'ember';
 import { moduleForComponent, test } from 'ember-qunit';
 
 moduleForComponent('tumblr-post', 'Unit | Component | tumblr post', {
-  // Specify the other units that are required for this test
-  // needs: ['component:foo', 'helper:bar'],
   needs: ['component:tumblr-post-text'],
   unit: true
 });
 
-test('it renders', function(assert) {
+test('it renders', function (assert) {
   assert.expect(2);
 
   // Creates the component instance
-  var component = this.subject();
+  const component = this.subject();
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).